### PR TITLE
BXMSDOC-1828 Update the RN for LA Release master/master-final

### PR DESCRIPTION
**Pull request by emmurphy1**
The rendered doc is [here|http://file.ork.redhat.com/~emmurphy/BXMSDOC-1838/].

### CODE REVIEW

File: docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+.Business Central home page
+The Business Central Home page has been redesigned for easier navigation. For more information, see the "Business Central" chapter of the _{PRODUCT} User Guide_.
+
+=== Guided Decision Table
+* Guided Decision Table has a new wizard for creating and editing columns.
+* Hit policies are now available. Hit policies determine the order in which rules (rows) in a guided decision table are applied.
+* New support to display multiple linked guided decision tables in the Guided Decision Table Editor has been added.
+
+=== Project Metrics Dashboard
+A new dashboard is now available for every project listed in the authoring library.
```
Comment (sutaakar) [inline docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc]: @emmurphy1 The old description of this feature was much longer. Was it reduced on purpose?

File: docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+=== Guided Decision Table
+* Guided Decision Table has a new wizard for creating and editing columns.
+* Hit policies are now available. Hit policies determine the order in which rules (rows) in a guided decision table are applied.
+* New support to display multiple linked guided decision tables in the Guided Decision Table Editor has been added.
+
+=== Project Metrics Dashboard
+A new dashboard is now available for every project listed in the authoring library. 
+
+=== Teams metrics dashboard
+A  new dashboard has been added to the *Teams* perspective.
```
Comment (sutaakar) [inline docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc]: Same as above.

File: docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+=== Process Execution Server
+The process execution server (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. 
+
+=== Separate workbench from execution server
+The workbench now delegates all of its requests to the execution server as well. The main advantage is that the workbench can now be used to monitor any set of execution servers. By linking the execution server to the workbench, the process and task monitoring UIs in the workbench can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+
+=== Smart router
+The smart router (also known as kie-server-router) can be used as a proxy to help manage multiple independent process execution servers.
+ 
+=== KIE Server Router enhancements
```
Comment (sutaakar) [inline docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc]: KIE Server Router is same component as Smart router, so this point could be merged with previous one. Considering that Smart router is a new component in 7.x I think this point could be skipped completely.

File: docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc (hunk 1 above)
Comment (emmurphy1) [reply docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc]: @sutaakar Yes, Christy asked me to summarize all of the entries.

File: docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc (hunk 3 above)
Comment (emmurphy1) [reply docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc]: @sutaakar removed section

**Pull request closed without merging**